examples/ror-with-kibana-reverse-proxy-demo/utils/collect-info-common.sh

```
@@ -0,0 +1,39 @@
+  version_1=$(echo "$1" | sed 's/-pre.*//')
+  version_2=$(echo "$2" | sed 's/-pre.*//')
+  [ "$version_1" = "$(echo -e "$version_1\n$version_2" | sort -V | tail -n 1)" ];
+}
+
+require_min_version() {
+  local provided_version="$1"
+  local min_version="$2"
+
+  if ! greater_than_or_equal $provided_version $min_version; then
+    echo "ERROR: ror-with-kibana-reverse-proxy-demo does not support Elasticsearch/Kibana versions lower than $min_version"
+    exit -3
```

Shellcheck (0.11.0)

[error] Can only exit with status 0-255. Other data should be written to stdout/stderr.

(SC2242)